pages/careers/_open-positions.html

```diff
@@ -237,5 +320,267 @@ <h4>Help us a create a fundamentally better DevOps experience!</h4>
       >
     </p>
 
+    <hr/>
+
+    <h2>Product</h2>
+
+    <h3 id="product-manager-iac" style="margin-top: 5px; margin-bottom: 5px">Senior Product Manager - Infrastructure as Code</h3>
+    <div style="background-color:#2e3b4a; padding:4px 6px; border-radius: 10px; display:inline-block; font-size:14px;"><strong>Time Zone:</strong> US time zones</div>
+
+    <h4>What You'll Work On</h4>
+    <ul>
+      <li>
+        <strong>Infrastructure as Code Library.</strong>
+        You will be the owner of the core of our business, the
+        <a href="/infrastructure-as-code-library" target="_blank">Infrastructure as Code (IaC) Library</a>,
+        a collection of over 350,000 lines of reusable, battle-tested, production-grade infrastructure code for AWS.
+        You'll be responsible for setting the direction for this library, which is written in Terraform, Go, Bash, and
+        Python and used in production by hundreds of companies.
+      </li>
+      <li>
+        <strong>Reference Architecture.</strong>
+        You'll also be the owner of the
+        <a href="/reference-architecture" target="_blank">Reference Architecture</a>, an opinionated, end-to-end tech
+        stack built on top of the Infrastructure as Code Library that we can deploy into a customer's AWS accounts in
+        about one day. You'll be responsible for defining the next generation of the Reference Architecture as part of
+        a major upcoming redesign.
+      </li>
+      <li>
+        <strong>Compliance.</strong>
```

Comment:
```suggestion
        <strong>Compliance Platform.</strong>
```

I think we have a bolder vision for compliance beyond just "here are some modules." So naturally it makes sense to introduce the SaaS du jour word of "platform." In truth, though, that is more reflective of how we've been thinking about it.

Reply:
Done: https://github.com/gruntwork-io/gruntwork-io.github.io/pull/710/commits/353d23d06048ef0e7573743f45c817c7148a9c6b